addons/website_twitter/views/res_config_settings_views.xml

```diff
@@ -30,21 +30,17 @@
                                 <blockquote class="small">
                                     <h2>How to configure the Twitter API access</h2>
                                     <ol>
-                                        <li>Create a new Twitter application on <a href="https://apps.twitter.com/app/new" target="new">https://apps.twitter.com/app/new</a>
-                                              with the following values:
+                                        <li>Log in or create an account on <a href="https://developer.twitter.com/" target="new"> https://developer.twitter.com/</a></li>
+                                        <li>Once connected, go to <a href="https://developer.twitter.com/portal/" target="new">https://developer.twitter.com/portal/</a></li>
+                                        <li>Complete the Twitter portal access process and <strong>create a project</strong> with the following information:
```

Comment:
> Complete the Twitter portal access process

I guess you only do that once? I don't see it in my account.

Maybe you could then rephrase it to something like "If not yet done, .." and split the rest in another step (the part about creating a project)
Because from the portal URL you gave, you can't create a project, it is at https://developer.twitter.com/en/portal/projects-and-apps

```suggestion
                                        <li>Once connected and if not already done, complete the Twitter portal access process on <a href="https://developer.twitter.com/portal/" target="new">https://developer.twitter.com/portal/</a></li>
                                        <li><strong>Create a project</strong> on <a href="https://developer.twitter.com/en/portal/projects-and-apps" target="new">https://developer.twitter.com/en/portal/projects-and-apps</a> with the following information:
```